### ISSUE TITLE
truncate_objects is not longer an array.

### ISSUE THREAD

**Issue by LiamKearn** [silverstripe/silverstripe-populate#42]
`truncate_objects` is no longer an array and seems to be causing issue for us.

I think this is related to the removal of the default in #32 .

**Comment by LiamKearn** [comment]
EG. https://github.com/silverstripe/silverstripe-populate/pull/32/files#diff-93dc79eb6bab3a17b4ea782351fd413f3b01e654949378c6b8dd974acdbc26ecR81 Foreach expects to iterate but doesn't get an iterable.

**Comment by davejtoews** [comment]
It's not an array because it's undefined unless you set a value in your `app.yml`. Ran into the same issue myself. If you don't want to truncate anything you can set an empty array in `app.yml` to prevent the error.

```yml
DNADesign\Populate\Populate:
  truncate_objects: []
```

**Comment by LiamKearn** [comment]
Thanks @davejtoews, should have moved to fix this when I discovered it!

This will make our CI a little cleaner if when we update next :)

**Comment by DanielTate** [comment]
>  This should be in the readme.

**Comment by DanielTate** [comment]
Is there a way to truncate only the objects created using the module?

**Comment by michalkleiner** [comment]
> Is there a way to truncate only the objects created using the module?

No. There's no record or a way of knowing which objects were created by this module and which existed before or after.

**Comment by wilr** [comment]
FIxed in latest release
